tests/test_features/test_projection.py

```diff
@@ -121,7 +122,7 @@ def test_discrete_plot(self):
         classes = ["a", "b", "c", "d", "e"]
         visualizer = MockVisualizer(projection=2, colormap="plasma", classes=classes)
         X_prime = visualizer.fit_transform(X, y)
-        assert(visualizer.classes_ == classes)
+        npt.assert_array_equal(visualizer.classes_, classes)
```

Comment:
Side effect of changing `classes_` to a np array.

Reply:
:+1: